apps/galleries_institutions/components/filter_facet/partner_filter_facet.coffee

```diff
@@ -35,9 +36,10 @@ module.exports = class PartnerFilterFacet extends Backbone.Model
     escape= (s) ->
       s.replace /[-\/\\^$*+?.()|[\]{}]/g, '\\$&'
 
+    query = AsciiFolder.fold(query)
```

Comment:
#superminor Instead of overwriting `query`, one option would be to declare a new variable `var foldedQuery = `.  I've found this can be easier to read in the future.